### ISSUE TITLE
no_strip option command-line

### ISSUE THREAD

**Issue by altert** [libvips/libvips#1879]
Is no_strip option available for command-line vips/vipsthumbnail tools? and if it is, what is correct format for it?

**Comment by jcupitt** [comment]
Hello @altert, it's just `strip` and you can use it as an option to any saver. For example:

```
vips copy x.jpg x.png[strip]
```

**Comment by altert** [comment]
thanks a lot! Your level of support for vips is something almost unimaginable)

**Comment by jcupitt** [comment]
Don't be impressed -- I'm having to so some very, very boring work for a deadline, and answering libvips questions is a pleasant distraction :(

**Comment by altert** [comment]
ok then it seems to be a good time to ask further) I'm trying to try out conversion adobeRGB jpeg to dzi while keeping color profile intact. Would be
`vips dzsave a.jpg zoom[strip]`  - be correct way to do it? (I'm using vips 8.10.2 built from source)

**Comment by jcupitt** [comment]
Ah OK, you need:

```
vips dzsave a.jpg zoom --no-strip
```

`dzsave` strips by default, since most people do not want metadata on every tile. Enter `vips dzsave` (no arguments) and press return to see a summary of all the options.

Have you considered conversion to sRGB instead? It would save a lot of storage and downloading for only a small reduction in colour gamut.

**Comment by altert** [comment]
Usually yes, but there are use cases when keeping color profile is a must.
I've tried `vips dzsave a.jpg zoom --no-strip`
but based on exiftool reporting it seems that color profile information is still lost

**Comment by jcupitt** [comment]
You're right, `--no-strip` did enable metadata on tiles, but not all metadata, only the default set. The source image metadata was not being copied down layers, so tiles were missing things like the original icc profile.

Thanks for reporting this dumb thing! The fix will be in 8.10.3.

I've added a test as well, and credited you in the changelog.

**Comment by jcupitt** [comment]
This is done, I'll close.
